packages/zostso/src/IssueTso.ts

```diff
@@ -73,63 +73,59 @@ export class IssueTso {
                 return response;
             } catch (e) {
                 if (e.message?.includes("status 404")) {
-                    // Set useNewApi to false to handle fallback logic
-                    useNewApi = false;
+                    // Continue to the old API behavior
                 } else {
                     // Re-throw for other exceptions
                     throw e;
                 }
```

Comment:
can we simplify this to...
```suggestion
                if (e.message && !e.message.includes("status 404")) throw e;
```

Reply:
Please look over the change. I made a small modification to keep the logic the same.

I implemented:
```
if (e.message == null || !e.message.includes("status 404")) throw e;
```